input/pagecontent/reference.md

```diff
@@ -871,14 +870,24 @@ For a path to a code within a code system, use this syntax:
   #active ^designation[0].value
   ```
 
+##### Caret Paths for Code Metadata {%include tu.html%}
```

Comment:
The example above (lines 867-871) is a caret path to a concept designation, so it needs to be moved to this section.  I think it would be good to move the designation example (rather than _remove_ it) because it was the most common use case people asked about when wanting to use `^` on codes.  So keep it, but move it to this section.

Reply:
done.